src/entry.c

```diff
@@ -257,19 +282,30 @@ static entry *entryWrite(char *buf,
 }
 
 /* Takes ownership of value. does not take ownership of field */
-entry *entryCreate(const_sds field, sds value, long long expiry) {
+entry *entryCreate(const char *field, size_t field_len, sds value, long long expiry) {
```

Comment:
why is the field being changed here? IIUC only the value can be "non-sds" but the fields will still be a regular sds right?